k8s/values.yaml

```diff
@@ -3,7 +3,7 @@ namespace: dino-park
 rev: latest
 replicas: 1
 features:
-  custom-invitation-text: false
-  history-tab: false
-  edit-group-type: false
-  access-groups-toggle: false
\ No newline at end of file
+  customInvitationText: false
```

Comment:
This name change must be reflected in `values/dev-preview.yaml` and `values/test-preview.yaml` too.